src/components/tables/Table.tsx

```diff
@@ -252,6 +253,7 @@ export class Table extends React.Component<ITableProps> {
                     handleOnRowClick={this.props.handleOnRowClick}
                     additionalRowClasses={this.props.additionalRowClasses}

```

Comment:
I think the prop `withoutHoverOnRow` and all the logic that goes with it is not necessary as it could be passed down using the `additionnalRowClasses`.

Reply:
I think the additionalRowClasses is for a custom behaviour. But in this case, its a case more common and its more simple to have a boolean prop to handle that